src/gameoflife.py

```diff
@@ -101,8 +111,8 @@ def main():
     args = parser.parse_args()
     
     # set iteration count
-    frames = 100
-    if args.frames and int(args.frames) > 100:
+    frames = 1000

```

Comment:
why start this at 1000 thats too many iterations

Reply:
just resetting back to what you submitted it originally to